docs/book/v1/intro.md

```diff
@@ -74,6 +74,20 @@ return [
         Grant\ImplicitGrant::class          => Grant\ImplicitGrant::class,
         Grant\RefreshTokenGrant::class      => Grant\RefreshTokenGrant::class
     ],
+
+    // optionally configure event listeners
+    // 'listeners' => [
```

Comment:
The listeners should be explained separately under an additional headline. The code examples shouldn't be added as code comments.